client/modules/summary.js

```diff
@@ -0,0 +1,36 @@
+import {
+  defaultSummary
+} from './defaults';
+
+const REQUEST_SUM = 'budgeting-sample-app/summary/REQUEST_SUM';
```

Comment:
I really like using these descriptive action values. 👍 
In my experience it gets difficult to track when strings are too long.  Can we shorten to something like `const REQUEST_SUM = 'budgeting/summary/REQUEST_SUM';`  or similar?

In a real world app, I would skip the `budgeting/` part, but this makes more sense here